### PR TITLE
Adding chat-falcon-7b to registry

### PR DESCRIPTION
**Pull request by biswaroop1547**
This PR helps integrate [falcon-7b service](https://github.com/premAI-io/prem-services/pull/13) to the registry.

### CODE REVIEW

File: chat-falcon-7b/README.md [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+!pip install langchain
+!pip install openai
+
+import os
+from langchain.chat_models import ChatOpenAI
+from langchain.schema import AIMessage, HumanMessage
+
+os.environ["OPENAI_API_KEY"] = "random-string"
+
+chat = ChatOpenAI(openai_api_base="http://localhost:8448/api/v1", max_tokens=128)
```
Comment (filippopedrazzinifp) [inline chat-falcon-7b/README.md:72]: Remove /api/

File: chat-falcon-7b/manifest.json [hunk 2]
```diff
@@ -0,0 +1,23 @@
+{
+    "id": "falcon-7b",
+    "name": "Falcon-7B",
```
Comment (filippopedrazzinifp) [inline chat-falcon-7b/manifest.json:3]: Remove `-` from the name

**Review by filippopedrazzinifp — COMMENTED** [review]
No need for any prompt template? 

Very good job in general :)

**Comment by biswaroop1547** [comment]
> No need for any prompt template?

Yes it was required, added changes

**Pull request merged**